docs/source/data/how_to_create_a_custom_dataset.md

```diff
@@ -1,7 +1,12 @@
-# Custom datasets
+# Advanced: Tutorial to create a custom dataset
```

Comment:
Not sure What's the problem. I try to do a search with "Custom Dataset" nothing relevant show up.

https://kedro--2888.org.readthedocs.build/en/2888/search.html?q=custom+dataset&check_keywords=yes&area=default#

Reply:
It's possible something to do with the RTD build. If you look at my build on github pages: https://stichbury.github.io/search.html?q=custom+dataset&check_keywords=yes&area=default